Config/bootstrap.php

```diff
@@ -4,10 +4,10 @@
 
 // For CakeDC search plugin
 Configure::write('Search', [
-		'Prg' => [
-			'commonProcess' => ['paramType' => 'querystring', 'filterEmpty' => true],
-			'presetForm' => ['paramType' => 'querystring']
-		]
+	'Prg' => [
```

Comment:
Indentation looks off.

Reply:
There were 2 tabs here. I removed one.

Reply:
Technically, 2 are correct :)
This is on 2nd nesting level.
Build the normalized form and you will see.